### DISCUSSION TITLE
Can't connect planetscale to Myql cli

### DISCUSSION THREAD

**Discussion by stebil87** [planetscale/discussion#350] [Q&A]
Hello
I've just started using planetscale. I am trying to connect it to mysql using mysql command line client.
I just copy-past and execute the command line I get on the web page. I don't get any error page on the shell but nothing happens on planetscale. What am I missing or doing wrong? Thanks

**Comment by jgreet** [comment]
Sorry, it's not clear what the problem is here. You run the `mysql -h ...` command to connect to the PlanetScale database and then what? The subsequent commands you run on the mysql command line don't work?

Could you show us the commands you're trying and the response you get back?

Thanks!

**Reply by stebil87** [reply]
Hello. Thanks. On the planetscale website there is no further explanation, as far as I could see, of these "subsequent commands" you are asking me for. What should I write after the -h... command? Thanks a lot.

**Reply by jgreet** [reply]
It's still not clear to me what you're trying to do or what you expect to happen.

After you connect the mysql CLI client to a new branch you could try creating and dropping a table to check if everything's working:
```
~$ mysql -h us-east.connect.psdb.cloud -u <redacted> -ppscale_pw_<redacted> --ssl-mode=VERIFY_IDENTITY --ssl-ca=/etc/ssl/cert.pem

mysql> create table t (id int, primary key (id));
Query OK, 0 rows affected (0.06 sec)

mysql> insert into t (id) values (123);
Query OK, 1 row affected (0.02 sec)

mysql> drop table t;
Query OK, 0 rows affected (0.05 sec)
```

If you want to see how branching and schema changes on PlanetScale work you could try following our Quickstart guide:
https://planetscale.com/docs/tutorials/planetscale-quick-start-guide

If you're trying to connect an application you're writing you'll want to use the MySQL driver for your language or framework and not the `mysql` CLI client. Maybe start with our guide for the language https://planetscale.com/docs

**Comment by stebil87** [comment]
Thanks. 
I am developing a webservice in python, this should be able also to manage a database in MySQL and a team of people should be able to remotely connect to this database. I need an online mysql-based database, editable from python. The webservice must be able to retrieve information from the database and update it when needed. 
Can planetscale do this? 
I hope this is clear.

**Reply by jgreet** [reply]
Yes, PlanetScale databases are MySQL compatible, hosted in the AWS region of your choice, and reachable via the internet. You can add your team members to the organization on PlanetScale and assign them access to the database:

https://planetscale.com/docs/concepts/access-control

To get an idea of the differences between PlanetScale and vanilla MySQL I'd recommend checking out our compatibility guide and taking a look at how branching and online schema changes work:

https://planetscale.com/docs/reference/mysql-compatibility
https://planetscale.com/docs/concepts/branching

We also have a blog post with configuration examples for several Python MySQL drivers:
https://planetscale.com/blog/connect-to-a-mysql-database-in-python
